### PR TITLE
[MB-16327] Update transportation offices

### PR DESCRIPTION
**Pull request by melissathai**
## Summary

When https://github.com/transcom/mymove/pull/11310 was merged, migrations on staging failed due to foreign key constraint violations when updating the `duty_locations` table ([link](https://app.circleci.com/pipelines/github/transcom/mymove/56673/workflows/9aa275e5-f081-4f7d-a270-b35bd9a75c66/jobs/972261)).

However, the `transportation_offices` migration seemed to work without issue. To help unblock this ticket, this PR only migrates the transportation offices as described in the spreadsheet linked in the above Jira ticket.

The Python script in this PR is essentially the same as was in Scott's [original PR](https://github.com/transcom/mymove/pull/11310), with some stylistic Pythonic tweaks.

## Verification Steps for the Author

These are to be checked by the author.

- [x] Verified that the SQL output in this PR is identical to https://github.com/transcom/mymove/pull/11310, except for new, randomly-generated uuids
- [x] Verified that the Contact box still shows up normally for service members

### How to test

1. Verify that the `transportation_offices` and `addresses` tables are in a valid state, and that there are 3 more TOs and addresses added after the migration is run
2. Verify that the new TOs show up as expected on the frontend (from the admin panel?)

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [x] Have been communicated to #g-database.

### CODE REVIEW

File: migrations/app/schema/20230906013850_update_transportation_offices.up.sql [hunk 1]
```diff
@@ -0,0 +1,888 @@
+BEGIN
```
Comment (rogeruiz) [inline migrations/app/schema/20230906013850_update_transportation_offices.up.sql:1]:
```suggestion
BEGIN;
```

**Review by rogeruiz — COMMENTED** [review]
From the look of the Python script, I see the semicolon in the print statement but don't see it in the generated file. Is this the latest output from the finalized Python file?

File: migrations/app/schema/20230906013850_update_transportation_offices.up.sql (hunk 1 above)
Comment (melissathai) [reply migrations/app/schema/20230906013850_update_transportation_offices.up.sql:1]: Originally the output had the semicolon, then I removed it when I saw other examples in the codebase that use `BEGIN` without the semicolon. But I forgot to remove it from the Python script!

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

Great job on this @melissathai. I'm glad that we were able to unblock at least half of this work. I don't have access to the original CSV data, but I did validate the SQL here and it passed.

I wanted to also add that I see the Python code was added to the `cmd/load-spreadsheet-data/`. I would have expected this to actually live in the `scripts/` directory with some updates to the README.md in that folder. But I don't feel strongly about where these Python scripts live nor where they are documented. I would like them to be documented though. Can we please add a follow-up PR where a README.md is dropped in the `cmd/load-spreadsheet-data/` directory? Or possibly consider migrating things over to another directory in another PR?

**Comment by melissathai** [comment]
That's a great point @rogeruiz . Honestly, the only reason why I put the script in `cmd/load-spreadsheet-data/` is because that's where it was in the [original PR](https://github.com/transcom/mymove/pull/11310). 

Is the `scripts/` directory the normal place where one-off scripts can live? Fwiw, the only [other one-off Python script](https://github.com/transcom/mymove/blob/main/cmd/load-spreadsheet-data/load-updated-loa-trnsn-ids.py) in the repo is also in the `cmd/load-spreadsheet-data/` directory, so if we move this script, should we move the other one too?

Happy to make documentation changes in a follow-up PR.

**Comment by melissathai** [comment]
Closing this PR because even though https://github.com/transcom/mymove/pull/11310 was reverted, it didn't rollback the TO migration (this was discovered during load testing). So merging this PR would cause redundant data to be created. Currently following up in #g-database on to best document the migration.

**Pull request closed without merging**